webrtc-stats.html

```diff
@@ -2049,6 +2050,21 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the target bitrate
```

Comment:
2.5 is a magic number, we should state somewhere that the number comes from an analysis, so that it would be easier to update the number by either re-analysing the data or by a number from another standard.

Reply:
Added a new paragraph justifying the magic number.